package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "mayflower",
+  "version": "1.0.0",
+  "repository": "https://github.com/massgov/mayflower.git",
+  "license": "UNLICENSED",
```

Comment:
We should probably match the license to GPLv2.  But also, can we set `private: true` in this file so it never accidentally gets published to NPM?